src/components/Pages/SetupPage/SetupPage.tsx

```diff
@@ -292,6 +292,7 @@ export const SetupPage = () => {
         </div>
         <div className={styles.right}>
           <WearablePreview lockBeta={true} panning={false} disableBackground={true} profile={profile} dev={false} />
+          <Loader active size="huge" />
```

Comment:
There is no need to hide the loader after the preview is loaded because the preview will just hide it. 

- code + happiness